languages/purescript/reference/errors/ErrorParsingFFIModule.md

```diff
@@ -0,0 +1,16 @@
+# `ErrorParsingFFIModule` Error
+
+## Cause
+
+This error occurs when the compiler cannot verify foreign declarations in JavaScript code,
+because it was unable to parse a JavaScript file.
+
+## Fix
+
+- Verify the JavaScript code in your foreign modules.
+- Foreign modules should conform to the ES5 specification. Values in foreign modules should be provided in the form `exports.name = value;`.
```

Comment:
This is the CommonJS module system (i.e. Node.js), not ES5 I believe

Reply:
That's ES5 syntax. It's helping the reader understand that using ES6 module syntax will not work:
```javascript
export const iAmTheBest = () { }
```

will fail if you try to import into PureScript.

Reply:
True, yes. It's more "not ES6" than it "is ES5". Without the CommonJS conventions it's just "ES5 assignment to a global object property". The rest of ES6 is allowed though, so it's the CommonJS import/export that's the requirement, not ES5 in general. This works, for example:

```purs
const doSomething = () => xyz();
exports.doSomething = doSomething;
```

Reply:
And CommonJS is convention on top of ES5 (and up), not part of ES5.

Reply:
Ah yes okay